Cargo.toml

```diff
@@ -30,7 +30,6 @@ libc = "0.2.72"
 
 [target.'cfg(windows)'.dependencies]
 winx = "0.19.0"
```

Comment:
Can we remove the `witx` dependency here?

Reply:
So we could, but given that we'll most likely need it similarly to the way we need `yanix` for debug printing I thought I was going to leave it in, rather than remove it now just to add it back in. Unless we won't need `winx` for that at all... OK, let's remove it for now, and add it in the future if need be.

Reply:
Ah, then perhaps we should figure out how to move some part of the debug printing into `cap-primitives` so that we don't need `yanix` here either :-). It'll still be a transitive dependency, but the separation will help us avoid accidentally adding too many implementation details to the outer `cap-std` crate.

Reply:
Excellent idea! If you don't mind though, leave it for a subsequent PR?